extensions/wikia/Chat2/Chat.class.php

```diff
@@ -658,12 +649,10 @@ public static function addConnectionLogEntry() {
 			$dbw->commit();
 		}
 
-		wfProfileOut( __METHOD__ );
 	}
 
-
-	static protected function getUserIPMemcKey( $userId, $address ) {
-		return $userId . '_' .  $address . '_v1';
+	protected static function getUserIPMemcKey( $userId, $address ) {
+		return wfSharedMemcKey( $userId, $address, 'v1' );
```

Comment:
Please add a prefix to this key so it's easier to identify it: eg. `wfSharedMemcKey( 'Chat', 'userIP', $userId, $address, 'v1' )`